src/app/components/card/card.component.scss

```diff
@@ -16,163 +16,160 @@
  * specific language governing permissions and limitations
  * under the License.
  */
+
+
+@import "variables";
+
+:host {
+    flex: 1 1 auto;
+    display: block;
+    max-width: 384px;
```

Comment:
Why 384px aha ?

Reply:
size that permit the dashboard to fit all with when we have a certain size screen. I ll try to find some thing more explicit